### PR TITLE
feat: more analytics

### PR DESCRIPTION
**Pull request by HarryET**
# Description
Includes requested additional analytics

Resolves # (issue)

## How Has This Been Tested?
Build locally

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/handlers/push_message.rs
Comment (xav) [inline src/handlers/push_message.rs]: Nice to have, but isn't it enough with just `status` in the table?

File: src/handlers/push_message.rs
Comment (xav) [inline src/handlers/push_message.rs]: Is the presence of 'response_message' the only differentiator between a request and a response?
In that case, can we have a field that tells explicitly which is which to avoid potential downstream issues?

File: src/handlers/push_message.rs
Comment (xav) [inline src/handlers/push_message.rs]: One more reason to remove this field. Someone someday will just do a count on 'success=false' and panic because there are a lot of failures :)

File: src/handlers/push_message.rs
Comment (xav) [inline src/handlers/push_message.rs]: Is that a third type of item? In that case, maybe a third type in addition to 'request' and 'response; could cover this?

**Review by xav — COMMENTED** [review]
Did you check with the data team already about these changes? You need a green light from them before changing the format/semantics of the data lake.

File: src/handlers/push_message.rs
Comment (dnul) [reply src/handlers/push_message.rs]: my intention with the success field was to give echo server the responsibility to determine when a request is successful or not, given it has more context on what's being done. Given it's just

>        message_info.success = status >= 200 && status < 300;

I don't mind dropping the column. 

Consider if in  the future  things becomes more complicated like `if status == 2xx AND some other condition` then it would be a problem for data to determine, with this column you control whether the outcome was successful or not.

**Review by arein — APPROVED** [review]

File: src/handlers/push_message.rs
Comment (HarryET) [reply src/handlers/push_message.rs]: response message was so we could provide context on errors

File: src/handlers/push_message.rs
Comment (HarryET) [reply src/handlers/push_message.rs]: Success is redundant as pointed out above, its been removed

File: src/handlers/push_message.rs
Comment (HarryET) [reply src/handlers/push_message.rs]: done

File: src/handlers/push_message.rs
Comment (HarryET) [reply src/handlers/push_message.rs]: I've removed it for now

File: src/handlers/push_message.rs
Comment (xav) [reply src/handlers/push_message.rs]: ok, I think we still need to be able to differentiate between requests and responses

**Comment by HarryET** [comment]
Builds should be all sorted now, waiting for CI & Re-review before merge + deploy

**Review by arein — APPROVED** [review]

**Pull request merged**
